docs/content/node-operation/full-observer-node.mdx

```diff
@@ -1,38 +1,221 @@
+```
+
+For testnet-37
+```
+wget -P bootstrap/public-root-information https://storage.googleapis.com/flow-genesis-bootstrap/testnet-37/public-root-information/root-protocol-state-snapshot.json
+wget -P bootstrap/public-root-information https://storage.googleapis.com/flow-genesis-bootstrap/testnet-37/public-root-information/root-protocol-state-snapshot.json.asc
+```
+
+#### Verify the PGP signature
+
+Add the `flow-signer@onflow.org` public key
+```
+gpg --keyserver keys.openpgp.org --search-keys flow-signer@onflow.org
+```
+
+Verify the root-snapshot file
+```
+gpg --verify bootstrap/public-root-information/root-protocol-state-snapshot.json.asc
+```
+
+```
+gpg: assuming signed data in 'bootstrap/public-root-information/root-protocol-state-snapshot.json'
+gpg: Signature made Wed Sep 15 11:34:33 2021 PDT
+gpg:                using ECDSA key 40CD95717AC463E61EE3B285B718CA310EDB542F
+gpg: Good signature from "Flow Team (Flow Full Observer node snapshot verification master key) <flow-signer@onflow.org>" [unknown]
+gpg: WARNING: This key is not certified with a trusted signature!
+gpg:          There is no indication that the signature belongs to the owner.
+Primary key fingerprint: 7D23 8D1A E6D3 2A71 8ECD  8611 CB52 64F7 FD4C DD27
+     Subkey fingerprint: 40CD 9571 7AC4 63E6 1EE3  B285 B718 CA31 0EDB 542F
+```
+
+Alternately, if you don't care about the blocks before the current block, you can request the current root-snapshot file via the flow-cli.
+
+For mainnet
+```
+flow snapshot save ./bootstrap/public-root-information/root-protocol-state-snapshot.json --network mainnet
+```
+
+For testnet
+```
+flow snapshot save ./bootstrap/public-root-information/root-protocol-state-snapshot.json --network testnet
+```
+
+### Step 4 Start the node
+
+The observer node can be run as a docker container
+
+#### Observer for Flow Mainnet
+
+```
+docker run --rm \
+ -v /var/flow/bootstrap:/bootstrap:ro  \
+ -v /var/flow/data:/data:rw \
+ --name flow_observer \
+ --network host  \
+ gcr.io/flow-container-registry/observer:v0.27.1 \
+ --bootstrapdir=/bootstrap \
+ --datadir=/data/protocol \
+ --bind 0.0.0.0:3569  \
+ --loglevel=info \
+ --secretsdir=/data/secrets  \
+ --upstream-node-addresses=access-008.mainnet18.nodes.onflow.org:9001 \
+ --upstream-node-public-keys=11742552d21ac93da37ccda09661792977e2ca548a3b26d05f22a51ae1d99b9b75c8a9b3b40b38206b38951e98e4d145f0010f8942fd82ddf0fb1d670202264a \
+ --bootstrap-node-addresses=access-008.mainnet18.nodes.onflow.org:3570  \
+ --bootstrap-node-public-keys=11742552d21ac93da37ccda09661792977e2ca548a3b26d05f22a51ae1d99b9b75c8a9b3b40b38206b38951e98e4d145f0010f8942fd82ddf0fb1d670202264a \
+ --observer-networking-key-path=/bootstrap/network.key
+```
+
+#### Observer for Flow Testnet
+
+```
+docker run --rm \
+ -v /var/flow/bootstrap:/bootstrap:ro  \
+ -v /var/flow/data:/data:rw \
+ --name flow_observer \
+ --network host  \
```

Comment:
I think it might be better to recommend people run with `-p 9000:9000` instead since `--network host` doesn't work on mac. There's also a [`--publish-all`](https://docs.docker.com/network/host/) that might work.

Reply:
removed the `--network host` and added `-p` for all ports.